hasura/migrations/default/1713188205470_alter_table_public_notifications_add_column_second_actor_profile_id/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table "public"."notifications" add column "second_actor_profile_id" int8
```

Comment:
seems a bit weird as a column name. i think we should re-rethink the DB schema and see how best to design this. @sshmm